config/routes.rb

```diff
@@ -2,6 +2,9 @@
   root 'users#index'
   
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :comments, only: [:create]
```

Comment:
There should be a pair of routes for creating a comment. However, in your case, you only have it for `create`, once you add `new` method in CommentController, you can add a route for it. It is one of the requirements for this project. ⬇️
![image](https://user-images.githubusercontent.com/33268927/216781138-c4179bf4-5cca-4588-b534-cfd44a534b43.png)

Reply:
@muneebulrehman  As mentioned in my previous comment, there is no need to add it to the routes since the user will not be redirected to a `comments/new` page to fill in a form. The form is already rendered on the post's page to allow the user to add comments without being redirected.